src/github.com/getlantern/flashlight/util/common.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	"errors"
+	"github.com/getlantern/golog"
+	"github.com/mitchellh/go-homedir"
```

Comment:
go-homedir is recommended when cross-compiling since it doesn't require the use of cgo on Darwin the way "os/user" does. Is this something we would want to use in byteexec as well?